babel.config.js

```diff
@@ -35,15 +37,7 @@ module.exports = function getBabelConfig(api) {
         },
       },
     ],
-    'babel-plugin-optimize-clsx',
-    [
-      '@babel/plugin-transform-runtime',
-      {
-        useESModules,
-        // any package needs to declare 7.4.4 as a runtime dependency. default is ^7.0.0
-        version: '^7.4.4',
-      },
-    ],
+    ['@babel/plugin-transform-runtime', { regenerator: false, version: '^7.26.10' }],
```

Comment:
in core/X we read the runtime version from the package.json

Reply:
:+1: updated